### PR TITLE
Append required user actions and only when the password is temporary

### PR DESCRIPTION
**Pull request by pkeuter**
Fixed the issues introduced in https://github.com/keycloak/keycloak/pull/27145

See (https://github.com/keycloak/keycloak/pull/27145/files/a4c12a7261a79d23a910cdf11c35e82d04d2271b)

### CODE REVIEW

**Review by edewit — DISMISSED** [review]

**Comment by pkeuter** [comment]
@edewit thanks for enabling auto-merge. I think a test is broken for some reason so I will have a look at that. You might to enable it again after that. Sorry!

**Comment by pkeuter** [comment]
@edewit this is now fixed

File: js/apps/admin-ui/src/user/UserForm.tsx [hunk 1]
```diff
@@ -82,7 +82,9 @@ export const UserForm = ({
   const [open, setOpen] = useState(false);
   const [locked, setLocked] = useState(isLocked);
 
-  setValue("requiredActions", user?.requiredActions || []);
+  useEffect(() => {
```
Comment (jonkoops) [inline js/apps/admin-ui/src/user/UserForm.tsx:85]: Why is this effect needed? Isn't the `user` already backing the form?

Comment (pkeuter) [reply js/apps/admin-ui/src/user/UserForm.tsx:85]: @jonkoops, I saw this one is still open. Do you need any additional information/changes or can I resolve the feedback?

Comment (jonkoops) [reply js/apps/admin-ui/src/user/UserForm.tsx:85]: Nope, just need to re-review the code here. Have some high prio things to take care of first.

Comment (pkeuter) [reply js/apps/admin-ui/src/user/UserForm.tsx:85]: Alright, I'll resolve this then. If you have any new feedback, I'll hear it :)

**Review by jonkoops — APPROVED** [review]

**Pull request merged**